lib/ansible/cli/__init__.py

```diff
@@ -8,6 +8,7 @@
 import locale
 import os
 import sys
+import functools
```

Comment:
This seems to be a leftover from a previous commit.

Reply:
Whoops! Fixing that.